alf/utils/value_ops.py

```diff
@@ -194,10 +193,8 @@ def one_step_discounted_return(rewards, values, step_types, discounts):
     discounts = discounts[1:]
     rewards = rewards[1:]
     values = values[1:]
-    step_types = step_types[:-1]
 
-    is_lasts = tf.cast(tf.equal(step_types, StepType.LAST), tf.float32)
```

Comment:
Why change this ? For `LAST step` , the value is  not  `reward + discount * V(FIRST step of new_episode)`,  it's `reward`. 
Thought it's masked out when calculate loss, it's better to use the correct calculation

Reply:
My bad. Will restore.